File: goals_calculator.py
import csv

with open('matches.csv', newline='') as f:
    reader = csv.DictReader(f)
    goals = [int(row['home_goals']) + int(row['away_goals']) for row in reader]

print(f"Average goals per match: {sum(goals)/len(goals):.2f}")

File: goals_calculator.js
const fs = require('fs');

// Get file path from command line or use default
const filePath = process.argv[2] || 'matches.json';

// Read and parse the file
const rawData = fs.readFileSync(filePath, 'utf-8');
const matches = JSON.parse(rawData);

// Calculate total goals and average
let totalGoals = 0;
let totalMatches = matches.length;

matches.forEach(match => {
  totalGoals += match.home_score + match.away_score;
});

const average = (totalGoals / totalMatches).toFixed(2);
console.log(`Average goals per match: ${average}`);